clients/rest/rest.go

```diff
@@ -123,6 +126,46 @@ func (c *Client) SetProxy(proxyUrl, user, password string) (err error) {
 	return
 }
 
+func (c *Client) SetCACerts(caFilePath string) (*Client, error) {
```

Comment:
May be this can take more than one ca to add multiple Intermediate CAs at once

Reply:
added!